lib/settings.py

```diff
@@ -200,8 +192,10 @@ def animation(text):
     global stop_animation
     i = 0
     while not stop_animation:
+        """
```

Comment:
Why?

Reply:
You realize that in order to stop the animation, I have to print a new line character otherwise it does an endless cycle